src/Data/Decimal.purs

```diff
@@ -238,3 +250,54 @@ foreign import gamma ∷ Decimal → Decimal
 factorial ∷ Decimal → Decimal
 factorial n | n < zero = one / zero
             | otherwise = gamma $ ceil (n + one)
+
+-- | Inverse secant.
+asec ∷ Decimal → Decimal
+asec x = acos (one / x)
+
+-- | Inverse cosecant.
+acsc ∷ Decimal → Decimal
+acsc x = asin (one / x)
+
+-- | Inverse cotangent.
+acot ∷ Decimal → Decimal
```

Comment:
Any suggestions for what to say about this function's domain in the function's doc comment?

Reply:
```purs
-- | Inverse cotangent. Note that this is a multivalued function. The definition here
-- | refers to the principal value of 'acot' without a discontinuity at x=0, and a domain
-- | of (0, π).
-- | See https://mathworld.wolfram.com/InverseCotangent.html for more information.
```